### ISSUE TITLE
Images get a pink background after uploading

### ISSUE THREAD

**Issue by mariannk** [nopSolutions/nopCommerce#3822]
nopCommerce version: 4.10

Steps to reproduce the problem: 
1. Go to product details page in admin area and upload a picture containing just white (#ffffff) background
2. Use any color picker to see that it is now #fffcff instead of #ffffff

More details at ticket 14874 (our premium support portal)

Also see related https://github.com/SixLabors/ImageSharp/issues/355 and https://github.com/SixLabors/ImageSharp/issues/502

**Comment by lulyhart** [comment]
I have struggled with the same problem. The source of the color change is related to the "default image quality" setting of 80.

The way to get around the color change is to set the quality to 100.
Configurations > Settings > Media > Default image quality (0 - 100) = 100

**Comment by mafdy** [comment]
It's a little bit of confusion.  

As I had struggled with the same problem, it was hard to understand the fundamental issue. When I set Image's quality to 100, image size raised to 1 MB which is a huge problem to any Store. About Images getting a pink background after uploading, it happens with me some time and then gone (without understand why). right now I had to upload images as jpg and 20 quality to prevent large images size (100 to 150 kb), and without getting a pink background it works for me.

nop version: 4.10 with source code.

hope this info can help.
Mafdy.
